client-side/C#/StreetSmartArcGISPro/StreetSmartArcGISPro/VectorLayers/VectorLayer.cs

```diff
@@ -932,8 +933,13 @@ private async Task ReloadSelectionAsync()
           {
             properties.Add(name, feature.GetOriginalValue(fieldId)?.ToString() ?? string.Empty);
           }
-          catch (NullReferenceException)
+          catch (Exception ex)
           {
+            if (!isLogged)
```

Comment:
Us this filed in specific only exception handler. The general one should log everything.